specification/iotcentral/data-plane/Microsoft.IoTCentral/preview/2019-10-28-preview/iotcentral.json

```diff
@@ -470,37 +399,43 @@
           "$ref": "#/definitions/SolutionModel"
         }
       },
-      "required": [
-        "types",
-        "capabilityModel"
-      ]
+      "required": ["types", "capabilityModel"]
     },
     "DeviceTemplateCollection": {
       "type": "object",
       "properties": {
         "value": {
           "description": "The collection of device templates.",
           "type": "array",
-          "items": {
-            "$ref": "#/definitions/DeviceTemplate"
-          }
+          "items": { "$ref": "#/definitions/DeviceTemplate" }
         },
         "nextLink": {
           "description": "URL to get the next page of device templates.",
           "type": "string"
         }
       },
-      "required": [
-        "value"
+      "required": ["value"]
+    },
+    "EmailUser": {
+      "allOf": [
+        { "$ref": "#/definitions/User" },
+        {
+          "type": "object",
+          "properties": {
+            "email": {
+              "description": "Email address of the user.",
+              "type": "string",
+              "format": "email"
```

Comment:
This is not a valid OpenAPI format and is the cause of the linter failure, can you please remove it.

Reply:
@jhendrixMSFT this _should_ work in theory
https://swagger.io/docs/specification/data-models/data-types/

> However, format is an open value, so you can use any formats, even not those defined by the OpenAPI Specification, such as:
> - email
> - uuid
> - uri
> - hostname
> - ipv4
> - ipv6
> and others
> Tools can use the format to validate the input or to map the value to a specific type in the chosen programming language. Tools that do not support a specific format may default back to the type alone, as if the format is not specified.

But I agree in practice the linter will not like it, and since it's ignored by autorest anyway, better to remove it

Reply:
I missed the part about format being open-ended, thanks for the correction.